src/test/java/com/nhnacademy/hello/controller/admin/CouponPageControllerTest.java

```diff
@@ -0,0 +1,120 @@
+package com.nhnacademy.hello.controller.admin;
+
+import com.nhnacademy.hello.common.feignclient.MemberAdapter;
+import com.nhnacademy.hello.common.util.AuthInfoUtils;
+import com.nhnacademy.hello.dto.member.MemberDTO;
+import com.nhnacademy.hello.dto.member.MemberStatusDTO;
+import com.nhnacademy.hello.dto.member.RatingDTO;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.MockitoAnnotations;
+import org.springframework.ui.Model;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+class CouponPageControllerTest {
+
+    @Mock
+    private MemberAdapter memberAdapter;
+
+    @Mock
+    private Model model;
+
+    private CouponPageController controller;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
@ExtendWith(Mockito.class)를 사용한다면 이것을 생략할 수 있지 않았을까요?
```suggestion
        MockitoAnnotations.openMocks(this);
```